packages/kbn-scout/src/playwright/fixtures/types/test_scope.ts

```diff
@@ -22,8 +22,30 @@ export interface LoginFixture {
   loginAsPrivilegedUser: () => Promise<void>;
 }
 
+/**
+ * Extends the Playwright 'Page' interface with methods specific to Kibana.
+ * Reasons to use 'ReturnType' instead of Explicit Typings:
+ * - DRY Principle: automatically stays in sync with the Playwright API, reducing maintenance overhead.
+ * - Future-Proofing: If Playwright changes the return type of methods, these types will update accordingly.
+ * Recommendation: define Explicit Types as return types only if methods (e.g. 'typeWithDelay')
+ * have any additional logic or Kibana-specific behavior.
+ */
```

Comment:
@wayneseymour I added more details in interface description. Hope it makes more sense. I think it requires to deep dive into [Playwright Library docs]( https://playwright.dev/docs/api/class-playwright) to get familiar with APIs and get most of the answers.